content/systems/filecoin_markets/onchain_storage_market/storage_market_actor.md

```diff
@@ -29,5 +37,5 @@ This collateral is returned to the storage provider when all deals in the sector
 
 
 {{<katex>}}
-$MinimumProviderDealCollateral = \\[0.2cm] \ \ \ \ \ \ \ \ 5\% \times FILCirculatingSupply \times \frac{DealRawByte}{max(NetworkBaseline, NetworkRawBytePower)}$
+$MinimumProviderDealCollateral = \\[0.2cm] \ \ \ \ \ \ \ \ 1\% \times FILCirculatingSupply \times \frac{DealRawByte}{max(NetworkBaseline, NetworkRawBytePower)}$
```

Comment:
This is not inline! Please read https://github.com/filecoin-project/specs#katex

Reply:
inline means inline with other content, there's no content around this formula. If you open the devtools you will see errors because [0.2cm] as no meaning when rendered in inline mode.

Reply:
you still using inline mode (`$math$`) when you should be using display mode (`$$math$$`)

Reply:
either way theres always inline and display mode, in this case where its a single formula with nothing around it, you want to highlight so it should be display mode `$$math$$`